### DISCUSSION TITLE
tutorials/growing-graphql-api-dgs-lab/02-adding-artist-type

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#114] [course feedback]
# tutorials/growing-graphql-api-dgs-lab/02-adding-artist-type

https://www.apollographql.com/tutorials/growing-graphql-api-dgs-lab/02-adding-artist-type

**Comment by mkatsman** [comment]
Mockup is good, but it would be helpful to have  a data type for each field, if I do not have to look at your code.
